src/main/java/com/exadel/discountwebapp/statistics/dto/RequestDataDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.exadel.discountwebapp.statistics.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class RequestDataDTO {
+    @NotNull
+    private String dataFrom;
```

Comment:
do you mean dateFrom? not dataFrom?
you can use LocalDateTime instead of String

Reply:
this class is not been used anymore. I've remove it